includes/polling/applications/bind.inc.php

```diff
@@ -2,20 +2,20 @@
 
 use LibreNMS\RRD\RrdDefinition;
 
-echo ' bind ';
-
+$name = 'bind';
+$app_id = $app['app_id'];
 if (!empty($agent_data['app'][$name])) {
     $bind = $agent_data['app'][$name];
 } else {
-    $name = 'bind';
-    $app_id = $app['app_id'];
     $options = '-O qv';
     $mib          = 'NET-SNMP-EXTEND-MIB';
     $oid     = 'nsExtendOutputFull.4.98.105.110.100';
     $bind    = snmp_get($device, $oid, $options, $mib);
 }
 update_application($app, $bind);
 
+echo " $name";
```

Comment:
Pop this back up the top, it needs to be echoed before we do anything for debugging.